ftml/src/render/html/element/collapsible.rs

```diff
@@ -87,78 +87,56 @@ pub fn render_collapsible(log: &Logger, ctx: &mut HtmlContext, collapsible: Coll
+                ))
                 .contents(|ctx| {
-                    // Event-bound link to open
+                    // Block is folded text
                     ctx.html()
-                        .a()
-                        .attr(attr!(
-                            "class" => "wj-collapsible-block-link",
-                            "href" => "javascript:;",
-                        ))
+                        .span()
+                        .attr(attr!("class" => "wj-collapsible-show-text"))
                         .inner(log, show_text);
+
+                    // Block is unfolded text
+                    ctx.html()
+                        .span()
+                        .attr(attr!("class" => "wj-collapsible-hide-text"))
+                        .inner(log, hide_text);
                 });
 
-            // Close collapsible link
+            // Content block
             ctx.html()
                 .div()
-                .attr(attr!("class" => collapsible_class(start_open)))
-                .contents(|ctx| {
-                    // Top div to close
-                    ctx.html()
-                        .div()
-                        .attr(attr!("class" => "wj-collapsible-block-unfolded-link"))
-                        .contents(|ctx| {
-                            ctx.html()
-                                .a()
-                                .attr(attr!(
-                                    "class" =>
-                                        "wj-collapsible-block-link "
-                                        collapsible_class(show_top),
-                                    "href" => "javascript:;",
-                                ))
-                                .inner(log, hide_text);
-                        });
-
-                    // Collapsed contents
-                    ctx.html()
-                        .div()
-                        .attr(attr!("class" => "wj-collapsible-block-content"))
-                        .inner(log, elements);
+                .attr(attr!("class" => "wj-collapsible-content"))
+                .inner(log, elements);
 
-                    // Bottom div to close
-                    ctx.html()
-                        .div()
-                        .attr(attr!("class" => "wj-collapsible-block-unfolded-link"))
-                        .contents(|ctx| {
-                            ctx.html()
-                                .a()
-                                .attr(attr!(
-                                    "class" =>
-                                        "wj-collapsible-block-link "
-                                        collapsible_class(show_bottom),
-                                    "href" => "javascript:;",
-                                ))
-                                .inner(log, hide_text);
-                        });
-                });
+            // Bottom open/close button
+            if show_bottom {
```

Comment:
`show_top` has different logic than `show_bottom`?

Reply:
The top button has to always be in the DOM because that button is how you control the collapsible when it's folded. CSS controls hiding the top button if it's not supposed to be visible.